packages/schemas/tables/sign_in_experiences.sql

```diff
@@ -15,5 +15,6 @@ create table sign_in_experiences (
   sign_in_mode sign_in_mode not null default 'SignInAndRegister',
   custom_css text,
   custom_content jsonb /* @use CustomContent */ not null default '{}'::jsonb,
+  mfa jsonb /* @use Mfa */,
```

Comment:
better give a not null default {} value?

Reply:
In my design, `null` means that MFA feature is not enabled.

Reply:
I changed my mind, updated.